src/test/resources/google/cloud/compute/v1/compute_strings.proto.baseline

```diff
@@ -0,0 +1,743 @@
+message Address {
+  // The type of address to reserve, either INTERNAL or EXTERNAL. If unspecified, defaults to EXTERNAL.
+  enum AddressType {
+    // A value indicating that the enum field is not set.
+    UNDEFINED_ADDRESS_TYPE = 0;
+
+    EXTERNAL = 35607499;
+
+    INTERNAL = 279295677;
+
+    UNSPECIFIED_TYPE = 53933922;
+
+  }
+
+  // The IP version that will be used by this address. Valid options are IPV4 or IPV6. This can only be specified for a global address.
+  enum IpVersion {
+    // A value indicating that the enum field is not set.
+    UNDEFINED_IP_VERSION = 0;
+
+    IPV4 = 2254341;
+
+    IPV6 = 2254343;
+
+    UNSPECIFIED_VERSION = 21850000;
+
+  }
+
+  // This signifies the networking tier used for configuring this address and can only take the following values: PREMIUM or STANDARD. Global forwarding rules can only be Premium Tier. Regional forwarding rules can be either Premium or Standard Tier. Standard Tier addresses applied to regional forwarding rules can be used with any external load balancer. Regional forwarding rules in Premium Tier can only be used with a network load balancer.
+  //
+  // If this field is not specified, it is assumed to be PREMIUM.
+  enum NetworkTier {
+    // A value indicating that the enum field is not set.
+    UNDEFINED_NETWORK_TIER = 0;
+
+    PREMIUM = 399530551;
+
+    STANDARD = 484642493;
+
+  }
+
+  // The purpose of this resource, which can be one of the following values:
+  // - `GCE_ENDPOINT` for addresses that are used by VM instances, alias IP ranges, internal load balancers, and similar resources.
+  // - `DNS_RESOLVER` for a DNS resolver address in a subnetwork
+  // - `VPC_PEERING` for addresses that are reserved for VPC peer networks.
+  // - `NAT_AUTO` for addresses that are external IP addresses automatically reserved for Cloud NAT.
+  enum Purpose {
+    // A value indicating that the enum field is not set.
+    UNDEFINED_PURPOSE = 0;
+
+    DNS_RESOLVER = 476114556;
+
+    GCE_ENDPOINT = 230515243;
+
+    NAT_AUTO = 163666477;
+
+    VPC_PEERING = 400800170;
+
+  }
+
+  // [Output Only] The status of the address, which can be one of RESERVING, RESERVED, or IN_USE. An address that is RESERVING is currently in the process of being reserved. A RESERVED address is currently reserved and available to use. An IN_USE address is currently being used by another resource and is not available.
+  enum Status {
+    // A value indicating that the enum field is not set.
+    UNDEFINED_STATUS = 0;
+
+    IN_USE = 17393485;
+
+    RESERVED = 432241448;
+
+    RESERVING = 514587225;
+
+  }
+
+  // IP address.
+  optional string I_p_address = 42976943;
+
+  // The static IP address represented by this resource.
+  optional string address = 462920692;
+
+  // The type of address to reserve, either INTERNAL or EXTERNAL. If unspecified, defaults to EXTERNAL.
+  // Check the AddressType enum for the list of possible values.
```

Comment:
I like the comment addition. 

Could you add a note to the PR description being explicit that we're purposefully still defining enum types, even if no fields are of those types?

Reply:
Sure, added.